decidim-proposals/spec/shared/proposal_form_examples.rb

```diff
@@ -61,15 +61,17 @@
   end
 
   context "when the title is too long" do
-    let(:body) { "A" * 200 }
+    let(:title) { "A" * 200 }
 
     it { is_expected.to be_invalid }
   end
 
-  context "when the body is not etiquette-compliant" do
-    let(:body) { "A" }
+  unless options[:skip_etiquette_validation]
```

Comment:
Is this really an option? I mean, it should sometimes be forced and other times it should not or it should never be forced?

Reply:
What do yo mean? I added this so we can use the shared examples both at admin and public forms (admin forms don't have the etiquette validation so we can skip the example).

Reply:
Yea, this is the answer to my question :smile: :+1: